### PR TITLE
Create ui-c8y-1021-27-0-plugins-installed-via-tag-should-also-correctly-show-up-in-the.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-60812](https://cumulocity.atlassian.net/browse/MTM-60812)
Original PR: [7828](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7828)

[MTM-60812]: https://cumulocity.atlassian.net/browse/MTM-60812?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1021-27-0-plugins-installed-via-tag-should-also-correctly-show-up-in-the.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: plugins installed via tag should also correctly show up in the list (#7828)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-27-0-plugins-installed-via-tag-should-also-correctly-show-up-in-the.md]: The Claude AI suggested to use the following title:
```suggestion
title: Plugins installed via tag are now correctly displayed in the list
```

File: content/change-logs/application-enablement/ui-c8y-1021-27-0-plugins-installed-via-tag-should-also-correctly-show-up-in-the.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60812
+version: 1021.27.0
+---
+plugins installed via tag should also correctly show up in the list (#7828)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-27-0-plugins-installed-via-tag-should-also-correctly-show-up-in-the.md]: The Claude AI suggested to use the following body:
```suggestion
Previously, plugins that were installed using a tag were not properly displayed in the list of installed plugins, which could cause confusion for users trying to manage their plugins. With this change, any plugin installed via tag will now correctly appear in the plugins list. This improvement enhances the accuracy and reliability of the plugin management system, ensuring that users have a clear overview of all installed plugins regardless of the installation method.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by janhommes — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1021-27-0-plugins-installed-via-tag-should-also-correctly-show-up-in-the.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60812
+version: 1021.27.0
+---
+Previously, plugins that were installed using a tag were not properly displayed in the list of installed plugins, which could cause confusion for users trying to manage their plugins. With this change, any plugin installed via tag will now correctly appear in the plugins list. This improvement enhances the accuracy and reliability of the plugin management system, ensuring that users have a clear overview of all installed plugins regardless of the installation method.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-27-0-plugins-installed-via-tag-should-also-correctly-show-up-in-the.md]:
```suggestion
Previously, plugins that were installed using a tag were not properly displayed in the list of installed plugins, which could cause confusion for users trying to manage their plugins. With this change, any plugin installed via a tag will now correctly appear in the plugins list. This improvement enhances the accuracy and reliability of the plugin management system, ensuring that users have a clear overview of all installed plugins regardless of the installation method.
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**